modules/optimization/src/optimizationreporters/ParameterMeshOptimization.C

```diff
@@ -46,30 +46,76 @@ ParameterMeshOptimization::validParams()
   params.addParam<unsigned int>(
       "num_parameter_times", 1, "The number of time points the parameters represent.");
 
-  params.addParam<std::vector<Real>>("initial_condition",
-                                     std::vector<Real>(),
+  params.addParam<std::vector<Real>>("constant_group_initial_condition",
                                      "Constant initial condition for each group of parameters.");
-  params.addParam<std::vector<Real>>(
-      "lower_bounds", std::vector<Real>(), "Constant lower bound for each group of parameters.");
-  params.addParam<std::vector<Real>>(
-      "upper_bounds", std::vector<Real>(), "Constant upper bound for each group of parameters.");
```

Comment:
add the old ones as deprecated parameters and either:
- make it still work
OR
- have an error message with instructions

Reply:
Can we just delete the old way of doing this since no one is using this module?

Reply:
if you're confident it s no one then yes I would be fine with it